docs/source/en/_toctree.yml

```diff
@@ -32,6 +32,8 @@
       title: Text-Guided Image-Inpainting
     - local: using-diffusers/depth2img
       title: Text-Guided Depth-to-Image
+    - local: using-diffusers/controlling_generation
+      title: Controlling generation
```

Comment:
Just a note: we have a mixture of title-casing and sentence-casing all over the place, for example `Text-Guided Depth-to-Image` but `Reusing seeds for deterministic generation`. Same thing for the headings inside the docs. A very minor thing, but at some point we need to standardize on something (/cc @yiyixuxu).